ur_controllers/src/gpio_controller.cpp

```diff
@@ -228,6 +242,9 @@ ur_controllers::GPIOController::on_activate(const rclcpp_lifecycle::State& /*pre
     safety_mode_pub_ =
         get_node()->create_publisher<ur_dashboard_msgs::msg::SafetyMode>("~/safety_mode", rclcpp::SystemDefaultsQoS());
 
+    program_state_pub_ =
+        get_node()->create_publisher<std_msgs::msg::Bool>("~/robot_program_running", rclcpp::SystemDefaultsQoS());
```

Comment:
It might make sense to make this latched.

Reply:
Not entirely sure, what is meant by latched, but I think i made it latched [here](https://github.com/urmahp/Universal_Robots_ROS2_Driver/blob/reset_controllers/ur_controllers/src/gpio_controller.cpp#L245-L248), if not let me know and then I will fix it.

Reply:
yes, exactly